includes/polling/unix-agent/packages.inc.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+$pkgs_id= array();
```

Comment:
a little picky, but could you change it to:
`pkgs_id = [];`
first, to match code guidelines, and initialize arrays in new style :-)

Reply:
Changed to match code guidelines.